packages/ui/src/views/secrets/create-secret/create-secret.tsx

```diff
@@ -247,6 +262,26 @@ export function CreateSecretPage({
               </Accordion.Content>
             </Accordion.Item>
           </Accordion.Root>
+
+          <Accordion.Root type="single" collapsible>
+            <Accordion.Item value="secret-manager">
+              <Accordion.Trigger>Secret Manager</Accordion.Trigger>
+              <Accordion.Content>
+                <ConnectorInput
+                  placeholder="Harness Built-in Secret Manager"
```

Comment:
this placeholder should not be hardcoded here. 
instead, the actual connector needs to be preselected.
placeholder will still be "Create or Select a Secret Manager".